### PR TITLE
docs for RTL support

### PR DESCRIPTION
**Pull request by grmmph**
Adding documentation for RTL support. It took me about an hour to figure out what's going on there.

### CODE REVIEW

**Comment by ericvicenti** [comment]
Thanks, this looks good enough to put in and iterate on later!

**Comment by mkonicek** [comment]
It looks like file names in /docs should have the .md extension and not contain spaces:
https://github.com/facebook/react-native/tree/master/docs

File: docs/Right-to-left support [hunk 1]
```diff
@@ -0,0 +1,35 @@
+## Right-to-left(rtl) support
+
+React native supports RTL languages. The best way to access RTL mode is via the `I18nManager`.
+
+### I18nManager
+
+#### I18nManager.isRTL()
+Getter, Returns boolean
+Checks if user's device is RTL
```
Comment (mkonicek) [inline docs/Right-to-left support]: Fullstop at the end of sentence.

File: docs/Right-to-left support [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+```
+const IS_RTL = I18nManager.isRTL();
+```
+
+#### I18nManager.allowRTL();
+Setter
+If true, If IS_RTL is `true` will change app layout to RTL
+If false, will disable RTL layout
```
Comment (mkonicek) [inline docs/Right-to-left support]: Fullstop at the end of sentence.

File: docs/Right-to-left support [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+#### I18nManager.isRTL()
+Getter, Returns boolean
+Checks if user's device is RTL
+
+```
+const IS_RTL = I18nManager.isRTL();
+```
+
+#### I18nManager.allowRTL();
+Setter
```
Comment (mkonicek) [inline docs/Right-to-left support]: "Setter" doesn't add much information.

File: docs/Right-to-left support [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+If true, If IS_RTL is `true` will change app layout to RTL
+If false, will disable RTL layout
+
+```
+const I18nManager.allowRTL(true); //
```
Comment (mkonicek) [inline docs/Right-to-left support]: Why '//'?

File: docs/Right-to-left support [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+```
+const I18nManager.allowRTL(true); // 
+```
+
+#### I18nManager.forceRTL(true);
+Setter
+If true, and even if user's device isn't RTL, will force app to use RTL layout
+
+```
+const I18nManager.forceRTL(true); //
```
Comment (mkonicek) [inline docs/Right-to-left support]: Why '//'?

**Comment by mkonicek** [comment]
There's this blogpost, would it make sense to link to it?
https://facebook.github.io/react-native/blog/2016/08/19/right-to-left-support-for-react-native-apps.html

**Comment by lacker** [comment]
It looks like this pull request has been abandoned. I'm going to close this but please feel free to reopen it if you would like to keep working on this!

**Pull request closed without merging**